tests/spec/component/versions.spec.js

```diff
@@ -25,38 +25,40 @@ if (process.platform === 'darwin') {
     describe('versions', function () {
         describe('get_tool_version method', () => {
             it('should not have found tool by name.', (done) => {
-                versions.get_tool_version('unknown').catch((error) => {
-                    expect(error).toContain('is not valid tool name');
-                    done();
-                });
+                versions.get_tool_version('unknown')
+                    .then(() => done(new Error('expected promise rejection')))
```

Comment:
I would favor using done.fail('...') instead of done(new Error(...)) throughout this test script, like this:

```suggestion
                    .then(() => done.fail('expected promise rejection'))
```

Reply:
Didn't know about that syntax. Let me give it a try...

Reply:
Looks good. Thanks!